src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -202,12 +231,27 @@ public synchronized ReadOnlyPerson addWhitelistedPerson(ReadOnlyPerson person) {
         return whitelistedPerson;
     }
 
+    /**
+     * Adds a specific person to overdue list in the AddressBook.
+     * @param person to be updated.
+     * @return overdueDebtPerson
```

Comment:
If I'm not wrong you need to elaborate a little on what overdueDebtPerson is in this @return line? @jelneo please help confirm

Reply:
will be safer to elaborate a bit about `overdueDebtPerson` even though the name is self-explanatory